src/makeCalculator.js

```diff
@@ -36,8 +36,32 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
-}
+  function makeCalculator() {
+    let obj = {
+      result: 0,
+      add: function(number) {
+        obj.result = (obj.result + number);
+      },
+      subtract: function(number) {
+        obj.result = (obj.result - number);
+      },
```

Comment:
```suggestion
      subtract(number) {
        obj.result -= number;
      },
```
![image](https://user-images.githubusercontent.com/96409563/207531456-dd712168-145c-4abc-971e-fec1969f579d.png)
![image](https://user-images.githubusercontent.com/96409563/207531502-b54f6bac-d738-4a25-a923-edf04a3bb964.png)